kitsune/messages/api.py

```diff
@@ -0,0 +1,113 @@
+from django.conf import settings
+
+from django.contrib.auth.models import Group, User
+from django.db.models import Q
+from django.views.decorators.http import require_GET
+
+from kitsune.access.decorators import login_required
+from kitsune.sumo.decorators import json_view
+from kitsune.sumo.utils import webpack_static
+from kitsune.users.models import Profile
+from kitsune.users.templatetags.jinja_helpers import profile_avatar
+
+
+def display_name_or_none(user):
+    try:
+        return user.profile.name
```

Comment:
Interesting - this was repeated from `users/api.py` so I assumed it was solving a problem. I'll remove and test. Thanks!

Reply:
If a user doesn't have a **profile** (which should never happen, but happens for me locally when I don't create a profile for a test user) - you don't get `None`, you get ye ole `Traceback....RelatedObjectDoesNotExist`. 

I think removing this *should* be fine in prod? Is there any case where we have production users without profiles? Even an invalid case would be bad if I remove this as it will break the return of suggestions by producing a 500 on the api call.